pmm-app/tests/verifyMysqlDashboards_test.js

```diff
@@ -1,59 +1,60 @@
 Feature('Test Dashboards inside the MySQL Folder');
+const allure = codeceptjs.container.plugins('allure');
 
 Before(async (I) => {
   I.Authorize();
 });
 
 Scenario(
-  // eslint-disable-next-line max-len
-  'Open the MySQL Overview Dashboard and verify Metrics are present and graphs are displayed @not-pr-pipeline',
-  async (I, adminPage, dashboardPage) => {
-    I.amOnPage(dashboardPage.mysqlInstanceSummaryDashboard.url);
-    dashboardPage.waitForDashboardOpened();
-    await dashboardPage.expandEachDashboardRow();
-    I.click(adminPage.fields.metricTitle);
-    adminPage.peformPageDown(5);
-    dashboardPage.verifyMetricsExistence(dashboardPage.mysqlInstanceSummaryDashboard.metrics);
-    await dashboardPage.verifyThereAreNoGraphsWithNA();
-    await dashboardPage.verifyThereAreNoGraphsWithoutData(3);
-  }
+    // eslint-disable-next-line max-len
+    'Open the MySQL Overview Dashboard and verify Metrics are present and graphs are displayed @not-pr-pipeline',
```

Comment:
missing severity

Reply:
i'm missing it on purpose because the default severity is `normal`

Reply:
yes realized it, no worries.